lib/canvas-motion.js

```diff
@@ -28,8 +30,14 @@ CanvasMotion.prototype = {
   }
 };
 
-function moveFishX(fish, round) {
-  fish.x += (fish.velocity * round);
+function moveFishX(fish, round, p5Amplitude) {
+  // console.log("moveFishX " + p5Amplitude);
+  fish.x += (fish.velocity * round/2 * p5Amplitude);
+  return fish;
+}
+
+function moveFishY(fish, p5Amplitude) {
+  fish.y = fish.y + p5Amplitude/150;
```

Comment:
sheisty math. Amplitude really isn't a factor here. We could use any method, but I do like how they sink off the screen very slowly